### DISCUSSION TITLE
Why are `.nomount` and `.nosync` files not checked for by default?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [bpozdena/OneDriveGUI#143] [Q&A] [answered]
As [`issues/141`](https://github.com/bpozdena/OneDriveGUI/issues/141#issue-2001196335) describes:

> I instructed `OneDriveGUI` to check for `.nomount` and `.nosync` files.

Why isn't this enabled by default?

**Comment by abraunegg** [comment, marked as answer]
> Why isn't this enabled by default?

Because it is a *client side* filtering rule that can be enabled by users - such as yourself. This is why it is not enabled by default are there are highly specific code operations (especially around `.nomount`) ... you have to understand *why* you are implementing that option + perform some very specific operating system modifications to use that option .... not everyone will be doing that ... thus not enabled by default.
